Cargo.toml

```diff
@@ -13,7 +13,7 @@ tracing-subscriber = { version = "0.3", features = ["env-filter"] }
 reqwest = { version = "0.11.10", features = ["json", "native-tls-vendored"] }
 url = "2.2.2"
 http = "0.2.7"
-
+clap = {version = "4.1.1", features = ["derive"] }
```

Comment:
this can come later but it would be nice if users can opt-in or opt-out of this feature

e.g. if someone only wants to use this as a library, they don't need to also grab `clap`

the way this is handled in Rust is to have sets of features, where some are optional

I have an example here: https://github.com/ralexstokes/ethereum-consensus/blob/main/Cargo.toml#L9

with a "default" feature set, which is "full featured"; the feature-level granularity lets users of this repo specify "no default features" to opt-out of the functionality they don't need

the question then here is: do we want the CLI thing to be default or not, and I'd say it is convenient enough to make it default, but behind a feature flag so that there is the "no default features" escape hatch